forge/forge/op/eval/forge/convolution.py

```diff
@@ -226,57 +226,47 @@ def eval(self, tensors):
         assert len(tensors) <= 3, "ConvTranspose ops should have up to three inputs (input, weight, bias)"
         assert len(tensors) >= 2, "ConvTranspose ops should have at least two inputs (input, weight)"
         t_ops = to_torch_operands(*tensors)
-
         activations = t_ops[0]
         weights = t_ops[1]
         bias = t_ops[2] if len(t_ops) == 3 else None
 
         stride = [self.stride_height, self.stride_width]
         dilation = [self.dilation_height, self.dilation_width]
         groups = self.groups
-        padding = [
-            self.padding_left,
-            self.padding_right,
-            self.padding_top,
-            self.padding_bottom,
-        ]
-
+        assert self.padding_top == self.padding_bottom, "Padding values for top and bottom must be the same."
```

Comment:
As this is a TTNN limitation, do we need to assert this here? 

Let's track this as their issues, and model it further if we see that this is by design :))

Reply:
@nvukobratTT , the TTNN limitation is for a different model i.e mobilenetv2 and not for this model, these changes has been added to modify the how padding has been handled in eval function of convtranspose2d op as a fix for monodle model.

Reply:
Is this an implementation limitation from our size regarding conv2d transpose? More precisely, my question is can we support this on FFE with different padding for each side? 

If yes, let's update assert message to be a bit more cleaner about why this is limitation :))

Reply:
Sure Nikola, I will make required changes.